specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -205,6 +205,511 @@
         "type": "object"
       }
     },
+    "ConnectorReadSetting": {
```

Comment:
Why are the type names here Setting vs Settings?

Reply:
Applies to all names

Reply:
I change "description": "Connector read settings."  to  "description": "Connector read setting."